charts/connaisseur/values.yaml

```diff
@@ -102,6 +102,9 @@ kubernetes:
   # (except for pods – see kubernetes.deployment.podLabels for that)
   additionalLabels: {}
 
+  # activate pod disruption budget
+  podDisruptionBudget: false
```

Comment:
How about exposing the minAvailable instead? Then the feature is more powerful and if someone wants to fiddle with it, we don't have to add to the API in a weird way?